lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua

```diff
@@ -47,6 +47,14 @@ e2function string entity:npcGetGlobalSquad()
     return squad
 end
 
+e2function number entity:npcGetCurrentSchedule()
+    if not isValidNpc( this ) then return 0 end
+    
+    local out = this:GetCurrentSchedule()
+    
+    return out
```

Comment:
A local variable like this is unnecessary especially if it just has a generic name like `out`
you can just do `return this:GetCurrentSchedule()`

Reply:
oh sure